packages/flutter/lib/src/rendering/animated_size.dart

```diff
@@ -363,6 +380,12 @@ class RenderAnimatedSize extends RenderAligningShiftedBox {
     }
   }
 
+  void _animationStatusListener(AnimationStatus status) {
+    if (status == AnimationStatus.completed) {
```

Comment:
This should be a switch per our style guide: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-using-if-chains-or--or--with-enum-values

Reply:
Done